Client/Application/ImageIdentifiers.swift

```diff
@@ -38,6 +38,7 @@ public struct ImageIdentifiers {
     public static let findInPage = "menu-FindInPage"
     public static let findNext = "find_next"
     public static let findPrevious = "find_previous"
+    public static let firefoxFavIcon = "faviconFox"
```

Comment:
nit, favicon is one word and not two. So no need of capital letter on the `i`
```suggestion
    public static let firefoxFavicon = "faviconFox"
```